packages/admin-ui/src/tag/tag.css.ts

```diff
@@ -0,0 +1,39 @@
+import type { Theme } from '@vtex/admin-ui-core'
+import { dataAttr, csx } from '@vtex/admin-ui-core'
+import { get } from '@vtex/admin-ui-util'
+
+function customPalette(token: string) {
+  return {
+    bg: `${token}10`,
+    color: '$primary',
+  }
+}
+
+export const tagTheme = csx({
+  display: 'inline-flex',
+  alignItems: 'center',
+  borderRadius: '100px',
+  color: '$primary',
+  border: 'none',
+  paddingX: '0.75rem',
+  outlineColor: (theme: Theme) => get(theme, 'bg.primary', 'bg.primary'),
```

Comment:
I mean, we could use a solid color here. Or import `theme` from `@vtex/admin-ui-core`